controlplane/pkg/nsmd/serviceregistry.go

```diff
@@ -241,6 +241,17 @@ func (impl *nsmdServiceRegistry) WaitForDataplaneAvailable(model model.Model, ti
 	st := time.Now()
 	for ; true; <-time.After(100 * time.Millisecond) {
 		if dp, _ := model.SelectDataplane(); dp != nil {
+
+			// Wait for mechanisms configuration
+			if !dp.RemoteConfigured {
```

Comment:
Is it possible to check `dp.RemoteMechanisms == nil` or `len(dp.RemoteMechanisms) == 0` instead of creating new flag `RemoteConfigured`?

Reply:
Dataplane may not have RemoteMechanisms after Mechanisms Update